LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2015 Workiva Inc.
+Portions Copyright 2016 The Chromium Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The existing content matches what the Apache License web page says to do. The instructions can be found at the bottom of:
http://www.apache.org/licenses/LICENSE-2.0